go.mod

```diff
@@ -6,14 +6,18 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.2.0
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0
 	contrib.go.opencensus.io/exporter/zipkin v0.1.1
-	github.com/cespare/reflex v0.2.0 // indirect
 	github.com/go-chi/chi v4.0.2+incompatible
-	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
+	github.com/gorilla/mux v1.7.3
 	github.com/micro/cli v0.2.0
-	github.com/ogier/pflag v0.0.1 // indirect
 	github.com/oklog/run v1.0.0
 	github.com/openzipkin/zipkin-go v0.2.2
 	github.com/owncloud/ocis-pkg v1.2.0
+	github.com/rs/zerolog v1.17.2
+	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/viper v1.6.1
+	github.com/stretchr/testify v1.4.0
 	go.opencensus.io v0.22.2
+	stash.kopano.io/kc/konnect v0.28.0
 )
+
+replace stash.kopano.io/kc/konnect v0.28.0 => /home/ilja/code/konnect
```

Comment:
you might want to remove this

Reply:
or replace it with his commit: `replace stash.kopano.io/kc/konnect v0.28.0 => github.com/IljaN/konnect v0.28.1-0.20200110083759-c7d0c1cc7dae` until the PR is merged

Reply:
indeed, adding the comment not to forget about it :D

Reply:
Will remove it when ready to merge.